aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -0,0 +1,180 @@
+
+Create a new ASP.NET Core app with the following command:
+
+```dotnetcli
+dotnet new web -o WebMetric
+cd WebMetric
+dotnet add package OpenTelemetry.Exporter.Prometheus.AspNetCore --prerelease
+dotnet add package OpenTelemetry.Extensions.Hosting
+```
+
+Replace the contents of `Program.cs` with the following code:
+
+:::code language="csharp" source="~/log-mon/metrics/metrics/samples/Program.cs":::
+
+<!-- Review TODO: Add link to available meters -->
+
+## View metrics with dotnet-counters
+
+[dotnet-counters](/dotnet/core/diagnostics/dotnet-counters) is a command-line tool that can view live metrics for .NET Core apps on demand. It doesn't require setup, making it useful for ad-hoc investigations or verifying that metric instrumentation is working. It works with both <xref:System.Diagnostics.Metrics?displayProperty=nameWithType> based APIs and [EventCounters](/dotnet/core/diagnostics/event-counters).
+
+If the [dotnet-counters](/dotnet/core/diagnostics/dotnet-counters) tool isn't installed, run the following command:
+
+```dotnetcli
+dotnet tool update -g dotnet-counters
+```
+
+While the test app is running, launch [dotnet-counters](/dotnet/core/diagnostics/dotnet-counters).
+The following command shows an example of `dotnet-counters` monitoring all metrics from the `Microsoft.AspNetCore.Hosting` meter.
+
+```dotnetcli
+dotnet-counters monitor -n WebMetric --counters Microsoft.AspNetCore.Hosting
+```
+
+Output similar to the following is displayed:
+
+```dotnetcli
+Press p to pause, r to resume, q to quit.
+    Status: Running
+
+[Microsoft.AspNetCore.Hosting]
+    http-server-current-requests
+        host=localhost,method=GET,port=5045,scheme=http                    0
+    http-server-request-duration (s)
+        host=localhost,method=GET,port=5045,protocol=HTTP/1.1,ro           0.001
+        host=localhost,method=GET,port=5045,protocol=HTTP/1.1,ro           0.001
+        host=localhost,method=GET,port=5045,protocol=HTTP/1.1,ro           0.001
+        host=localhost,method=GET,port=5045,protocol=HTTP/1.1,ro           0
+        host=localhost,method=GET,port=5045,protocol=HTTP/1.1,ro           0
+        host=localhost,method=GET,port=5045,protocol=HTTP/1.1,ro           0                                            12
+```
+
+Run `WebMetric>dotnet-counters list` to show all available metrics.
+
+The following command shows the `Microsoft-AspNetCore-Server-Kestrel` meter with the `requests-per-second` and `total-requests` counters.
+
+```dotnetcli
+dotnet-counters monitor -n WebMetric --counters Microsoft.AspNetCore.Hosting[requests-per-second,total-requests]
+```
+
+For more information, see [dotnet-counters](/dotnet/core/diagnostics/dotnet-counters). To learn more about metrics in .NET, see [built-in metrics](/dotnet/core/diagnostics/available-counters).
+
+## Create custom metrics
+
+See [Create custom metrics](/dotnet/core/diagnostics/metrics#create-custom-metrics) for information on creating custom metrics.
```

Comment:
There is no 'Create-custom-metrics' heading on that page. This should probably be https://learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics-instrumentation#create-a-custom-metric